hal/armv7m/imxrt/10xx/console.c

```diff
@@ -72,10 +83,26 @@ void console_init(void)
 	imxrt_dataBarrier();
 
 	/* Set 115200 baudrate */
-	t = *(halconsole_common.uart + uart_baud);
-	t = (t & ~(0x1f << 24)) | (0x4 << 24);
-	*(halconsole_common.uart + uart_baud) = (t & ~0x1fff) | 0x8b;
-	*(halconsole_common.uart + uart_baud) &= ~(1 << 29);
+	t = *(halconsole_common.uart + uart_baud) & ~((0x1f << 24) | (1 << 17) | 0x1fff);
+
+	/* For baud rate calculation, default UART_CLK=80MHz was assumed */
+	switch (
+#if CONSOLE_BAUD(UART_CONSOLE)
+		CONSOLE_BAUD(UART_CONSOLE)
+#else
+		BAUD(UART_BAUDRATE)
+#endif
```

Comment:
Maybe define another constant on top of the file and use it here? #if..#else...#endif inside of switch condition looks really bad

Reply:
Done.